web/package.json

```diff
@@ -42,6 +43,7 @@
     "http-status-codes": "^2.2.0",
     "ical.js": "^1.5.0",
     "katex": "^0.16.4",
+    "lint-fix": "^0.24.0",
```

Comment:
Why was this added?

Reply:
I think I ran `npm i lint-fix` instead of `npm run lint-fix`, sometimes my brain does wired stuff! Thanks for the catch ! 👍